hive-metastore/src/main/java/org/apache/iceberg/hive/CachedClientPool.java

```diff
@@ -132,7 +131,7 @@ public <R> R run(Action<R, IMetaStoreClient, TException> action, boolean retry)
   static Key extractKey(String cacheKeys, Configuration conf) {
     // generate key elements in a certain order, so that the Key instances are comparable
     List<Object> elements = Lists.newArrayList();
-    elements.add(conf.get(HiveConf.ConfVars.METASTOREURIS.varname, ""));
+    elements.add(conf.get("hive.metastore.uris", ""));
```

Comment:
It might be better to define this constant in an Iceberg util class.

Reply:
I think it is not necessary and I don't see the benefit. Spark had to deal with this issue too, and they used the conf name or `HiveConf.getConfVars("...")` with the conf name if they needed a enum instance. They didn't go so far as to define a collection of constants for the conf names.